src/WaveClip.h

```diff
@@ -216,24 +220,46 @@ class AUDACITY_DLL_API WaveClip final : public XMLTagHandler
 
    void SetColourIndex( int index ){ mColourIndex = index;};
    int GetColourIndex( ) const { return mColourIndex;};
-   void SetOffset(double offset);
-   double GetOffset() const { return mOffset; }
+   
+   double GetSequenceStartTime() const noexcept;
+   void SetSequenceStartTime(double startTime);
+   double GetSequenceEndTime() const;
+   sampleCount GetSequenceStartSample() const;
+   sampleCount GetSequenceEndSample() const;
+   sampleCount GetSequenceSamplesCount() const;
+
+   double GetPlayStartTime() const noexcept;
+   void SetPlayStartTime(double time);
+
+   double GetPlayEndTime() const;
+
+   sampleCount GetPlayStartSample() const;
+   sampleCount GetPlayEndSample() const;
+   sampleCount GetPlaySamplesCount() const;
+
+   void SetTrimLeft(double trim);
+   double GetTrimLeft() const noexcept;
+
+   void SetTrimRight(double trim);
+   double GetTrimRight() const noexcept;
```

Comment:
Comment that these four get and set the non-negative length of hidden material at each end